src/parser/PropertyPath.cpp

```diff
@@ -21,6 +21,17 @@ PropertyPath PropertyPath::makeModified(PropertyPath child,
   }
 }
 
+// _____________________________________________________________________________
+PropertyPath PropertyPath::makeModified(PropertyPath child,
+                                        std::string_view modifier, 
+                                        int64_t stepsMin, 
+                                        int64_t stepsMax) {
+  // TODO-Sven: Not sure if AD_CORRECTNESS_CHECK is needed. 
+  //            Should we be doing this with ANTLR4? {int, int} is handled, no?
+  //            But the rules are not set up for stepsMin/Max as INTEGER_POSITIVE.
```

Comment:
The cleanest way is to here do
CORRECTNESS_CHECKS (both positive, max > min),
And then in the SparqlQleverVisitor do proper checks (using the `reportError` function).